bower.json

```diff
@@ -27,13 +27,17 @@
   ],
   "dependencies": {
     "polymer": "Polymer/polymer#0.8-preview",
-    "core-icon": "Polymer/core-icon#0.8-preview",
-    "core-icons": "Polymer/core-icons#0.8-preview",
-    "core-signals": "Polymer/core-signals#0.8-preview",
+    "iron-icon": "polymerelements/iron-icon^0.8.0",
```

Comment:
All lowercase works, but `PolymerElements` just to be consistent with other org naming.